src/com/lftechnology/java_training/jan7/FileCreator.java

```diff
@@ -0,0 +1,126 @@
+package com.lftechnology.java_training.jan7;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.logging.Level;
+
+/**
+ * Write a program to create a text file in the path firstprogramfile.txt and check whether that file is exists. Using the command exists(),
+ * isDirectory(), isFile(), getName() and getAbsolutePath().
+ * 
+ * @author UtsabNeupane<utsabneupane@lftechnology.com>
+ * 
+ */
+public class FileCreator {
+	private static TrainingConstantsUtility constantValues = new TrainingConstantsUtility();
+
+	public static void main(String[] args) {
+		String path = "firstprogramfile.txt";
+		File newFile = new File(path);
+		createFile(newFile);
+		constantValues.LOGGER.log(Level.INFO, "Following are the property of file:\n1.{0} \n2.{1} \n3.{2} \n4. its absoulte path is: {3} ",
```

Comment:
@anishmanandhar to display the output like this:
Following are the property of file:
1.file exists 
2.this file is not a directory 
3.this is a file 
4. its absoulte path is: /home/utsabn34/codehome/repo/java_training/firstprogramfile.txt 
 i have done so dai how to do it in alternative way??